script/Deploy.s.sol

```diff
@@ -1,17 +1,54 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.24;
 
-import {Script, console2} from "forge-std/Script.sol";
+import "@openzeppelin/proxy/transparent/TransparentUpgradeableProxy.sol";
+import "@openzeppelin/proxy/transparent/ProxyAdmin.sol";
+
+import "forge-std/Script.sol";
+import "forge-std/console.sol";
+
 import "../src/BatchInbox.sol";
 
 contract Deploy is Script {
     function setUp() public {}
 
     function run() public {
+        uint256 deployerPrivateKey = vm.envUint("PRIVATE_KEY");
```

Comment:
One question: the deployment requires a private key, which is hot in memory.  Can foundry support a deployment using a cold wallet?